tests/diamond.sh

```diff
@@ -101,8 +101,12 @@ tmux send-keys -t $session:$first_idx '../wmediumd/wmediumd -c diamond.cfg' C-m
 node_idx=$((first_idx + 4))
 tmux send-keys -t $session:$node_idx 'iperf -s' C-m
 
-# enable monitor
-tmux send-keys -t $session:$first_idx 'ip link set hwsim0 up' C-m
+# enable monitor and capture traffic as normal user so that capture file is not
+# owned by root
+tmux new-window -t $session
+cap_idx=$((first_idx + 5))
+tmux send-keys -t $session:$cap_idx 'ip link set hwsim0 up' C-m
+tmux send-keys -t $session:$cap_idx "sudo -u $SUDO_USER dumpcap -i hwsim0" C-m
```

Comment:
Hmm, $SUDO_USER assumes sudo was used to start the script, but it could have been started via a normal root shell.  I guess it's OK but it would be nicer to check whether it is set.